<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mathModelGenerationWindow</class>
 <widget class="QMainWindow" name="mathModelGenerationWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>542</width>
    <height>634</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Math model generation</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="4" column="0" rowspan="2" colspan="3">
     <widget class="QGroupBox" name="groupBox_3">
      <property name="title">
       <string>SLD</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label_12">
         <property name="font">
          <font>
           <family>Consolas</family>
          </font>
         </property>
         <property name="text">
          <string>sld = np.ones_like(x)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="textEdit_sld">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>75</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Consolas</family>
          </font>
         </property>
         <property name="html">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Consolas'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;sld = 6.34 * sld&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0" colspan="3">
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="3" column="0" colspan="2">
        <widget class="QLineEdit" name="lineEdit_modelName"/>
       </item>
       <item row="3" column="2" rowspan="4" colspan="3">
        <widget class="QTableView" name="tableView_params"/>
       </item>
       <item row="1" column="3">
        <widget class="QPushButton" name="pushButton_addParam">
         <property name="text">
          <string>Add</string>
         </property>
        </widget>
       </item>
       <item row="1" column="4">
        <widget class="QPushButton" name="pushButton_deleteParam">
         <property name="text">
          <string>Delete selected</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Model name</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Parameters</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QComboBox" name="comboBox_coord">
         <property name="font">
          <font>
           <family>3ds</family>
          </font>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Coordination</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="5">
        <widget class="QLabel" name="label_5">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Use only letter, number, and underline in model name and parameter name</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0" colspan="2">
        <widget class="QLabel" name="label_grid2xyz">
         <property name="font">
          <font>
           <family>Consolas</family>
          </font>
         </property>
         <property name="text">
          <string>x, y, z = grid_points</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0" colspan="2">
        <widget class="QLabel" name="label_4">
         <property name="font">
          <font>
           <family>Consolas</family>
          </font>
         </property>
         <property name="text">
          <string>import numpy as np</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="3" column="0" colspan="3">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Shape</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Use &quot;&amp;&quot;, &quot;|&quot; as &quot;and&quot;, &quot;or&quot;; use parentheses for each statement</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="textEdit_shape">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>75</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Consolas</family>
          </font>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="0" colspan="3">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Boundary</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_11">
         <property name="font">
          <font>
           <family>Consolas</family>
          </font>
         </property>
         <property name="text">
          <string>boundary_min, boundary_max = np.array([-1, -1, -1]), np.array([1, 1, 1])</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="textEdit_boundary">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>75</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Consolas</family>
          </font>
         </property>
         <property name="html">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Consolas'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;boundary_max = 1 * boundary_max&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;boundary_min = -1 * boundary_max&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="6" column="2">
     <widget class="QPushButton" name="pushButton_saveModel">
      <property name="text">
       <string>Save model</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
